docs/install/usb-install.md

```diff
@@ -15,6 +15,8 @@ There are a couple of ways to create a USB installation flash drive.
 :::caution
 **Known Issue:** For the v1.2.0 ISO image, there is a known [issue](https://github.com/harvester/harvester/issues/4510) where the interactive ISO installation will get stuck using the USB method.
 
+Now you can use the patchd [ISO](https://releases.rancher.com/harvester/v1.2.0/harvester-v1.2.0-patch1-amd64.iso) to resolve this issue. We only repack v1.2.0 ISO with the correct lable, and there is no other change. Also, you can use this sha512 [file](https://releases.rancher.com/harvester/v1.2.0/harvester-v1.2.0-patch1-amd64.iso.sha512sum) to verify the ISO.
```

Comment:
```suggestion
Now you can use the patchd [ISO](https://releases.rancher.com/harvester/v1.2.0/harvester-v1.2.0-patch1-amd64.iso) to resolve this issue. We only repack the v1.2.0 ISO with the correct label and nothing else is changed. Also, you can use this sha512 [file](https://releases.rancher.com/harvester/v1.2.0/harvester-v1.2.0-patch1-amd64.iso.sha512sum) to verify the ISO.
```

Reply:
One more suggestion... (sorry for missing this one earlier)

Reply:
Updated, thanks!